subsys/net/ip/Kconfig.mgmt

```diff
@@ -38,6 +38,15 @@ config NET_MGMT_EVENT_QUEUE_SIZE
 	  notification. Thus the size of this queue has to be tweaked depending
 	  on the load of the system, planned for the usage.
 
+config NET_MGMT_EVENT_QUEUE_TIMEOUT
+	int "Timeout for event queue"
+	default 10
+	range 1 10000
+	help
+	  Timeout in milliseconds for the event queue. If the queue is full,
+	  the oldest event will be removed and the new one will be added.
```

Comment:
Is this true? In cursory look it seems that if the `k_msgq_put()` fails in `mgmt_push_event()`, then the latest one is discarded, not the oldest one.

Reply:
True, nothing is removed in case of a timeout - the new event is discarded.